account_invoice_picking_address/account.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2015 Agile Business Group <http://www.agilebg.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    delivery_address_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Delivery Address',
+        compute='_compute_delivery_address')
+    sale_order_ids = fields.Many2many(
+        'sale.order', 'sale_order_invoice_rel', 'invoice_id', 'order_id',
```

Comment:
On 18 September 2015 at 11:25, Alex Comba <notifications@github.com> wrote:

> Just attribute comodel_name is mandatory so you could remove the others as
> optional.
>
>
>
This is the inverse of 'invoice_ids' field of 'sale.order', so I need them